blivet/flags.py

```diff
@@ -77,6 +77,9 @@ def __init__(self):
         # (so far only for LUKS)
         self.discard_new = False
 
+        # wheteher cdroms should be made protected during devicetree population
+        self.protect_cdroms = True
```

Comment:
Any reason why this should be configurable? Let's not introduce flags for absurd/artificial cases.

Reply:
The reason is that to be safe - I wanted to apply the change only to image installations (or better - installations on running system in the future).
This approach is definitely quite ad-hoc solution.
See the discussion in related anaconda PR: https://github.com/rhinstaller/anaconda/pull/1977
Maybe we could set OpticalDevices as protected by default?
I was also thinking the devices could be protected as being read-only (if _mark_readonly_device was not limited to disks but was applied also for cdroms) but apparently cdrom device is not read-only.